api/src/test/java/org/pmiops/workbench/billing/FreeTierBillingServiceTest.java

```diff
@@ -16,11 +16,14 @@
 import com.google.cloud.bigquery.LegacySQLTypeName;
 import com.google.cloud.bigquery.Schema;
 import com.google.cloud.bigquery.TableResult;
-import java.io.InputStream;
```

Comment:
Note: a lot of this is refactoring how we mock BigQuery, and you may want to review that piece separately.  This refactoring is its own commit: 849e6c7141a4abd26052569f51